lib/cloudflare_rails.rb

```diff
@@ -1,7 +1,9 @@
 require 'zeitwerk'
```

Comment:
is there any advantage to just renaming this file to `cloudflare-rails.rb`?

Reply:
That should be fine.

Reply:
i'll wait and try in a different pr, i'd like to get this out asap. thanks for your help!